test/testcafe/spec/Identity_spec.js

```diff
@@ -45,4 +53,19 @@ test(`should have correct display texts`, async t => {
 
   const needhelpLinkText = identityPage.getNeedhelpLinkText();
   await t.expect(needhelpLinkText).eql('Need help signing in?');
+});
+
+test(`should show global error for invalid user`, async t => {
+  await t
```

Comment:
we could remove `requestHooks` from `fixture` and change test to
`test(mock)(`should ...`, () => {})`

Reply:
This is also one way of doing it, so wanted to keep multiple ways of doing that.

Reply:
why `wanted to keep multiple ways of doing that.` ?

Reply:
Useful to be aware of, I didn't knew earlier it was possible.

Reply:
I see. usually I'd prefer consistency. (apparently I wasn't aware either..)